### ISSUE TITLE
Why is the 'change' to the weights impacted by the last activation?

### ISSUE THREAD

**Issue by fredthedead** [leriomaggio/deep-learning-keras-tensorflow#9]
In **1.1 Introduction - Deep Learning and ANN.ipynb**  there's this line: 

`change = output_deltas[k] * self.ah[j]`

and change is then being used to the update the output weights.

`self.wo[j][k] += N*change +  M*self.co[j][k]`

Why are the activations from the hidden layer being used to tweak the weights? shouldn't the weight changes be only based on the weights and the relative change of the loss?

**Comment by leriomaggio** [comment]
Oh well..`change` is the weight update, multiplied by the `eta` (i.e. `N`, the learning rate).
